README.md

```diff
@@ -20,43 +20,40 @@ composer require craftgate/craftgate
 ```
 
 ### Manual Installation
-You need to download the latest [release](https://github.com/craftgate/craftgate-php-client/releases) and copy to your project. Then, include the bootstrap file as shown below. This file will autoload all the class files into your project.
+You need to download the latest [release](https://github.com/craftgate/craftgate-php-client/releases) and copy to your project. Then, include the autoload file as shown below. This file will autoload all the class files into your project.
 
 ```php
-require_once('/path/to/craftgate-php-client/CraftgateBootstrap.php');
+require '/path/to/craftgate-php-client/autoload.php';
 ```
 
 ## Usage
 To access the Craftgate API you'll first need to obtain API credentials (e.g. an API key and a secret key). If you don't already have a Craftgate account, you can signup at [https://craftgate.io/](https://craftgate.io)
 
-Once you've obtained your API credentials, you can start using Craftgate by instantiating a `Craftgate` with your credentials.
+Once you've obtained your API credentials, you can start using Craftgate by instantiating a `Craftgate\Client` with your credentials.
 
 ```php
+use Craftgate\Client;
 
-$options = new RequestOptions();
-$options->setApiKey('<YOUR API KEY>');
-$options->setSecretKey('<YOUR SECRET KEY>');
-
-$craftgate = new Craftgate($options);
-...
-
+$client = new Client(array(
```

Comment:
Tüm clientlarda bir standartımız var ve class isimleri uyumludur. Bu yüzden eski halinin kalması bizim için önemli.

Reply:
https://github.com/craftgate/craftgate-php-client/pull/10#discussion_r569607694

Reply:
Done: e46e321